react/features/conference/components/native/styles.js

```diff
@@ -40,11 +41,11 @@ export default {
     },
 
     gradientStretchBottom: {
-        height: GRADIENT_HEIGHT + FILMSTRIP_SIZE
+        height: BOTTOM_GRADIENT_HEIGHT
     },
 
-    gradientStretch: {
-        height: GRADIENT_HEIGHT
+    gradientStretchTop: {
+        height: DEFAULT_GRADIENT_SIZE
```

Comment:
why default and not top? (name)

Reply:
Because that's the ```minHeight``` of the bottom gradient as well

Reply:
Ah, okidoki!